contracts/Octobay.sol

```diff
@@ -360,6 +366,59 @@ contract Octobay is Ownable, ChainlinkClient, BaseRelayRecipient {
 
 
 
+    // ------------ GOVERNANCE ------------ //
+
+
+    OctobayGovernor public octobayGovernor;
+    OctobayGovTokenFactory public octobayGovTokenFactory;
+
+    struct NewGovernanceToken {
+        bool isValue;
+        string githubUserId;
+        string name;
+        string symbol;
+        string projectId;
+        uint16 newProposalReq;
```

Comment:
I think the min quorum is missing here.
```solidity
uint16 minQuorum;
```

Reply:
Actually at the moment I've got the quorum required configurable per proposal instead of universal for the Governor. With the thinking that when you create more important proposals they need a larger share to pass... ?